### ISSUE TITLE
Make alt text mandatory

### ISSUE THREAD

**Issue by diesocke** [bluesky-social/social-app#4155]
**Is your feature request related to a problem? Please describe.**

Blind or visually impaired people are dependant on alt texts to understand images. Bluesky has a function to let the user be reminded of missing alt texts. However, one has to become aware of the problem that visually impaired people cannot interpret images without alt texts.

**Describe the solution you'd like**
Reverse the default setting of "Require alt text before posting" to set.

**Describe alternatives you've considered**
There are numerous people commenting skeets lacking alt text with hints to set this preference manually. As people often are unaware that the are excluding visually impaired people this leads to missunderstanding and conflict.

**Additional context**

**Comment by t0b1hh** [comment]
Yep. Default should be inverted. 

But a user should still be able to disable the requiredment of alt text.  

There are also people who are not able to describe images. They should still be able to post images even w/o alt text.

**Comment by cooljeanius** [comment]
Yeah, I think this should be retitled, as what this issue is actually asking for (inverting the default of the "Require alt text before posting" setting) is different from making it mandatory, IMO. I would support inverting the default, but not making it mandatory, though.

**Comment by chris-gds** [comment]
Did this get added by any chance? On mobile when I created a new account this was on by default (which had me excited) but when I did the same on desktop it wasn't on.

**Comment by LooneyH** [comment]
I'm hoping this is something that gets implemented, most people don't change default settings and this would help incentivise people to start adding _something_.
